### PR TITLE
Update vulkan readme notes

### PR DESCRIPTION
**Pull request by dj2**

### CODE REVIEW

**Comment by dj2** [comment]
@alan-baker does this seem like enough, or should I put some text around it as well?

File: README.md [hunk 1]
```diff
@@ -105,6 +105,10 @@ A Vulkan implementation is found by CMake in the following priority order:
      Example: `VULKAN_SDK=$HOME/vulkan-macos-1.1.85.0/macOS`
    * `VK_ICD_FILENAMES` should point to the ICD JSON file.
      Example: `VK_ICD_FILENAMES=$VULKAN_SDK/etc/vulkan/icd/MoltenVK_icd.json`
+   * `VK_LAYER_PATH` should point to the etc/explicit\_layer.d folder in your
+     vulkan SDK directory
+   * `LD_LIBRARY_PATH` must contain the lib/ folder for the validation
```
Comment (jaebaek) [inline README.md]: "`LD_LIBRARY_PATH` must contain `$VULKAN_SDK/lib/`"
would be more clear for this?

File: README.md [hunk 2]
```diff
@@ -105,6 +105,10 @@ A Vulkan implementation is found by CMake in the following priority order:
      Example: `VULKAN_SDK=$HOME/vulkan-macos-1.1.85.0/macOS`
    * `VK_ICD_FILENAMES` should point to the ICD JSON file.
      Example: `VK_ICD_FILENAMES=$VULKAN_SDK/etc/vulkan/icd/MoltenVK_icd.json`
+   * `VK_LAYER_PATH` should point to the etc/explicit\_layer.d folder in your
```
Comment (jaebaek) [inline README.md]: "`VK_LAYER_PATH` should point to `$VULKAN_SDK/etc/explicit_layer.d`"
would be more clear for this?

File: README.md (hunk 1 above)
Comment (dj2) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (dj2) [reply README.md]: done

**Pull request merged**